src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const total = dayRent * days;
+  const threeDayDiscount = total - 20;
+  const sevenDayDiscount = total - 50;
```

Comment:
2 points:
- `total - <value>` is not a discount, `<value>`, i.e. 20 or 50, are discounts
- the checklist, p.5